packages/cli/README.md

```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
+Android application.
+
+Developers who want to change their application configuration after running `init` can do so by
+editing this file then running the `update` command.
+
+Fields:
+
+|Name|Type|Required|Description|
+|:--:|:--:|:------:|:---------:|
+|packageId|string|true|The [application id](https://developer.android.com/studio/build/application-id) for the output Android app.|
+|host|string|true|The origin that will be opened in the Trusted Web Activity.|
+|name|string|true|Name for the Android application, displayed on vairous places when installed on an Android device.|
+|launcherName|string|false|A short name for the Android application, displayed on the Android launcher|
+|display|`'standalone'` \| `'fullscreen'` \| `'fullscreen-sticky'`|false|The initial display mode for the Android application. Defaults to `standalone`.|
+|themeColor|string|true|The color used for the status bar.|
+|navigationColor|string|true|The color used for the navigation bar.|
+|navigationColorDark|string|false|The color used for the navigation bar when the device is in dark mode. Defaults to `#000000`.|
+|navigationDividerColor|string|false|The color used for the navigation bar divider. Defaults to `#000000`.|
+|navigationDividerColorDark|string|false|The color used for the navigation bar divider when the device is dark mode. Defaults to `#000000`.|
+|backgroundColor|string|true|Color used for the splash screen background|
+|enableNotifications|boolean|true|Set to true to enable notification delegation.|
+|startUrl|string|true|The start path for the TWA. Must be relative to the domain.|
+|iconUrl|string|true|Full URL to an the icon used for the application launcher and splash screen. Must be at least 512x512.|
+|maskableIconUrl|string|false|Full URL to an the icon used for maskable icons, when supported by the device.|
+|monochromeIconUrl|string|false|Full URL to a monochrome icon, used when displaying notifications.|
+|splashScreenFadeOutDuration|number|true|Duration for the splash screen fade out animation.|
+|signingKey|[SigningKeyInfo](#SigningKeyInfo)|true|Information on the signing keys used to sign the output Android application. Read the [SigningKeyInfo](#SigningKeyInfo) section for details.|
+|appVersionCode|number|false|`versionCode` for the Android application. Check the [Android docs](https://developer.android.com/studio/publish/versioning) for details.|
+|appVersion|string|false|`versionName` for the Android application. Check the [Android docs](https://developer.android.com/studio/publish/versioning) for details.|
+|shortcuts|[ShortcutInfo](#ShortcutInfo)[]|false||
+|generatorApp|string|false|Identifier for tool used to generate the Android project. Bubblewrap uses `bubblewrap-cli`. Should only be modified by generator apps.|
+|webManifestUrl|string|false|Full URL to the PWA Web Manifest. Required for the application to be compatible with Chrome OS devices.|
+|fallbackType|`'customtabs'` \| `'webview'`|false|`customtabs` or `webview`|
```

Comment:
Description here could use a few more words.

Reply:
Yep, was pending that. Added.